libraries/engage/components/RangeSlider/index.jsx

```diff
@@ -0,0 +1,310 @@
+import React, { Component, Fragment } from 'react';
```

Comment:
I've experienced some issues with the slider. First of all it seems that the aria-role isn't fully implemented into iOS and Android. When focusing the the slider handles, the voice readers on both operating systems tell you how to operate with it (iOS: swipe up and down with one finger / Android: use volume keys to adjust). Doing so has no effect.

I found a closed webkit bug report that described the issue ([link](https://bugs.webkit.org/show_bug.cgi?id=174898)). It seems that APIs are missing to connect the screen reader with the webview.

[This comment](https://github.com/w3c/aria-practices/issues/8#issuecomment-493412418) from a ARIA practices repo on GitHub links to a codepen with a hack for the issue on iOS. But on Android it doesn't really work.

The iOS double tap solution to move the slider seems to work reliable on iOS. But I don't think, that screen reader users are used to such precise gestures. When moving the handles on iOS, users are not informed about updates. So they will not be able to understand that they are currently changing the price range. This could be solved by adding `aria-live="assertive"` to the `Label` component of the `PriceSlider`.

On Android i didn't manage to change the slider values.  

Maybe a solution with two hidden text field that are only visible for the screen reader would be better.

Reply:
Changed it to hidden inputs.